websocket.go

```diff
@@ -42,6 +42,11 @@ func NewWebSocket(si *network.ServerIdentity) *WebSocket {
 	webHost, err := getWebAddress(si, true)
 	log.ErrFatal(err)
 	w.mux = http.NewServeMux()
+	w.mux.HandleFunc("/ok", func(w http.ResponseWriter, r *http.Request) {
+		log.Lvl2("ok?", r.RemoteAddr)
+		ok := []byte("ok\n")
+		w.Write(ok)
+	})
```

Comment:
I don't know much about websockets, so let me if what I'm not making sense. Is this a standard thing to do? If it's not, do we need to write somewhere that prevents the user from making a service called "ok" to avoid conflicting handlers?

Reply:
Sounds good to have a fool's guard in `registerService`.